error-handler.js

```diff
@@ -98,15 +99,16 @@ var mixIn = require('mout/object/mixIn'),
     stream.pipe(res);
   },
 
-  send = function send(statusCode, err, res, o) {
+  send = function send(statusCode, err, res, o) {    
+    if (!err.message) err.message = statusCodes[statusCode];
```

Comment:
How about this?

```
send = function send(statusCode, err, res, o) {    
  var base = {
    status: statusCode
  };
  var extended = {};

  if (!err.message) base.message = statusCodes[statusCode];
  if (err.name) extended.name = err.name;

  var body = (o.serializer) ?
    o.serializer(mixIn({}, err, base, extended)) :
    base;

  res.status(statusCode);
  res.send(body);
}
```

IMO, hardcoding a key whitelist is more responsible than blindly mutating the `err` object, particularly since we're just talking about what is being sent to the user. The full error can be inspected and logged elsewhere by other middleware for other purposes.